tests/suites/model/migration.sh

```diff
@@ -401,21 +401,3 @@ test_model_migration_saas_external() {
 		run "run_model_migration_saas_external"
 	)
 }
-
-bootstrap_alt_controller() {
```

Comment:
is this unused?

Reply:
no, as this function is used in several tests: `cmr`, and `migration`,  there is a suggestion to move it to the `includes` folder.